src/librbd/ManagedLock.h

```diff
@@ -36,11 +36,22 @@ class ManagedLock {
               const std::string& oid, Watcher *watcher);
   virtual ~ManagedLock();
 
-  bool is_lock_owner() const;
+  bool is_lock_owner(bool exclusive) const;
```

Comment:
Minor: I'd move this down to the private section because the bool param is confusing / has a potential for future error